docs/standard/microservices-architecture/microservice-ddd-cqrs-patterns/domain-events-design-implementation.md

```diff
@@ -147,19 +148,18 @@ public abstract class Entity
 }
 ```
 
-When you want to raise an event, you just add it to the event collection to be placed within an aggregate entity method, as the following code shows:
+When you want to raise an event, you just add it to the event collection from code at any method of the aggregate-root entity, as the following code shows which is part of the [Order agregate-root at eShopOnContainers](https://github.com/dotnet-architecture/eShopOnContainers/blob/dev/src/Services/Ordering/Ordering.Domain/AggregatesModel/OrderAggregate/Order.cs):
```

Comment:
I suggest moving the last part of the sentence to a new sentence after the code example

Reply:
👍